.github/workflows/ci_linting.yml

```diff
@@ -24,14 +25,22 @@ jobs:
         continue-on-error: true
         with:
```

Comment:
Should you update line 24 to use cache@v3 as well?

Reply:
This isnt actually ready. Ill update the description with whats left and what the blockers are.